### ISSUE TITLE
Unicode with FontAwesome not work on IOS 14.

### ISSUE THREAD

**Issue by mikayilsrt** [xamarin/Xamarin.Forms#12407]
### Description
I have a Xamarin.Forms application using FontAwesome icons, it worked very well until I switched to IOS 14 which now displays the icons with ?
On Android the icons still work without any problem.

### Steps to Reproduce

1. Run xamarin.forms on IOS 14
2. 

### Expected Behavior
Icons (unicode) must be displayed on IOS in the same way as on Android.

### Actual Behavior
Icons on IOS are displayed with a "?".

### Basic Information

- Version with issue: Xamarin.Forms 4.8.0.1451
- IDE: Visual Studio for Mac (8.7.8)
- Platform Target Frameworks: 
  - iOS: IOS 14
- Affected Devices: IOS

### Screenshots
<img width="692" alt="Capture d’écran 2020-10-08 à 11 47 36" src="https://user-images.githubusercontent.com/9454583/95442867-2f4abb00-095c-11eb-84c4-a04988b86964.png">

**Comment by jsuarezruiz** [comment]
Mnn, the Mail icon is rendering correctly, use FontAwesome?.

Could you attach an example with the Unicode you are using and it doesn't render correctly?

**Comment by mikayilsrt** [comment]
@jsuarezruiz Hi,
Indeed I use Font Awesome, some icons work but not all on IOS 14.
I created a new test project and the problem is still there. You can download the sample here.

[TestUnicodeForms.zip](https://github.com/xamarin/Xamarin.Forms/files/5347327/TestUnicodeForms.zip)

**Comment by mikayilsrt** [comment]
Hi, I update this issue.

I solved my problem. I've removed the fa-brands-400.ttf and fa-regular-400.ttf and the references on the Info.plist
I think the issue is that these 2 files do have the same font name, and this conflicts in the rendering, but I'm not sure.

For the project I published as a zip file, I followed this post https://devblogs.microsoft.com/xamarin/embedded-fonts-xamarin-forms/ which also solved the problem but only for the test project I published above.
